test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -69,6 +69,26 @@ TEST_F(AwsLambdaFilterTest, DecodingHeaderStopIteration) {
   EXPECT_EQ(Http::FilterHeadersStatus::StopIteration, result);
 }
 
+TEST_F(AwsLambdaFilterTest, HostHeaderSanitized) {
+  setupFilter({arn_, InvocationMode::Synchronous, true /*passthrough*/});
+  TestScopedRuntime scoped_runtime;
```

Comment:
I think CI failed due to missing header
```
#include "test/test_common/test_runtime.h"
```
and corresponding deps in BUILD file

```
"//test/test_common:test_runtime_lib"
```

Reply:
@reskin89 you might need to include
`"//test/test_common:test_runtime_lib"`
 in https://github.com/envoyproxy/envoy/blob/main/test/extensions/filters/http/aws_lambda/BUILD#L21